about.html

```diff
@@ -0,0 +1,154 @@
+        </div>
+
+        <div id="about-contact">
+            <p class="about-description">Please contact us per Email for any further questions about FLAVORS OF PERU 2022.</p>
+            <a href="#" id="about-email">contact@flavorsofperu22.com</a>
+        </div>
+
+    </header>
+
+    <section id="about-logo">
+
+        <h2 id="about-logo-info" class="about-section-title">The FLAVORS OF PERU summit logo</h2>
+
+        <hr class="about-hr">
+
+        <p class="about-text-content">The logo of FLAVORS OF PERU 2022 is property of MARCA PERU and the office of tourism and foreign commerce of the Ministry of Tourism. </p>
+
+
+        <div class="about-logo-container">
+
+            <img src="images/feature-images/event-logo-black.png" class="event-logo" alt="Flavors of Peru logo">
+
+        </div>
+
+    </section>
+
+    <section id="about-past-events">
+
+        
+
+        <h2 id="about-join-info" class="about-section-title">See the past Flavors of Peru editions</h2>  
+
+        <hr class="about-hr hr-past-event">
+
+        <p class="about-text-content">Take a look at the last two Flavors of Peru summits which took place in Moscow and Lisbon.</p>
+
+
+        <div id="events-container">
+            <div id="moscow-lisbon">
+                <div class="lisbon-lisbon">    
+                    <h2 class="event-img-year inside-text">2019</h2>
+                    <h3 class="event-img-location inside-text">Flavors of Peru Lisbon</h3>
+                </div>
+
+                
+                <div class="moscow-moscow">    
+                    <h2 class="first-text">2018</h2>
+                    <h3 class="second-text">Flavors of Peru Moscow</h3>
+                </div>
+            </div>
+        </div>
+
+    </section>
+
+    <section id="partners">
+        <div class="partners-container">
+            <h2 class="footer-title">Partner</h2>
+            <hr class="about-hr">
+            <div class="img-partners-container">
+                <img class="partner-logo" src="images/logos/50best.png" alt="partner 50 best restaurants logo">
+                <img class="partner-logo" src="images/logos/bulgari.png" alt="partner Bulgari logo">
+                <img class="partner-logo" src="images/logos/klm.png" alt="partner KLM logo">
+                <img class="partner-logo" src="images/logos/marriot.png" alt="partner Marriot logo">
+                <img class="partner-logo" src="images/logos/moet.png" alt="partner MOET CHAMPAGNE logo">
+            </div>
+        </div>
+        </section>
+    
+        <footer class="page-footer">
+        
+            <div class="copyright">
+    
+                <img class="copyright-img" src="images/feature-images/marca-peru.png" alt="marca Peru logo">
+                <div id="rights">
+                    <p class="copyright-text">Marca Peru</p>
+                    <p class="copyright-text">All rights reserved</p>
```

Comment:
- [ ] This should be part only in mobile version and disappear in the desktop version
![Screenshot from 2021-10-14 22-47-41](https://user-images.githubusercontent.com/62705947/137393198-c744bf9f-8374-4743-b72b-9c745d202349.png)